mocks/uid.go

```diff
@@ -4,13 +4,16 @@ import (
 	"github.com/pborman/uuid"
 )
 
-// uuid.UUID is a slice and cannot be used as a map key.
+// UID is defined since uuid.UUID is a slice and cannot be used as a map key.
 type UID [16]byte
 
+// Interface converts the fixed length array representation to the uuid package
+// type.
 func (u *UID) Interface() uuid.UUID {
 	return uuid.UUID(u[:])
 }
 
+// IsNIL returns true if the UID value is 0.
```

Comment:
unrelated question: I can't understand this function.. it looks like returns always false.